LDM-556.tex

```diff
@@ -158,18 +158,18 @@ \subsubsection{Subsetting a DataRepository without data transfer}
 \textbf{ID:} DMS-MWBT-REQ-0010 (Priority: 1a)
 
 \textbf{Specification:}
-It shall be possible to easily create a new DataRepository which is a view of a sub-section of an existing DataRepository, given a list of DataUnits and a list of DatasetTypes.
+It shall be possible to easily create a new DataRepository which is a view of a sub-section of an existing DataRepository, given a list of DataCoordinates and a list of DatasetTypes.
```

Comment:
Here again (and in more places below), was this an instance of `DataRepository` that was missed, or is the distinction intentional?